arangod/Views/LoggerView.cpp

```diff
@@ -35,6 +35,8 @@
 
 using namespace arangodb;
```

Comment:
this is outside the scope of the task #344

Reply:
@vasiliy-arangodb The fact of presence `LoggerView` requires a useless changes in `LoggerView.h` and `LoggerView.cpp` to compile ArangoDB successfully which means that it's in the scope of the task. Since `LoggerView` is no longer needed please remove `LoggerView.*` files instead of changing them.